apps/style/applab/skins/modern.scss

```diff
@@ -30,16 +26,39 @@
   select {
     -webkit-appearance: none;
     -moz-appearance: none;
-    &::-ms-expand {
-      display: none;
-    }
-
+    border: 0;
     border-radius: 0;
     padding: 0 30px 0 10px;
     color: #fff;
     background: url("/blockly/media/applab/dropdown.png") no-repeat center right;
+
+    &::-ms-expand {
+      display: none;
+    }
+
     &:hover {
       background: url("/blockly/media/applab/dropdown.png") no-repeat center right, url("/blockly/media/applab/brighter.png");
     }
   }
+
+  button, select, input, [contenteditable=true] {
+    outline: 0;
+  }
+}
+
+#divApplab.appModern.divApplabDesignMode {
+
+  img[src=''], canvas {
+    border: 5px solid #fff;
+    outline: 1px solid #bdc3c7;
+    outline-offset: -1px;
+  }
+
+  img[src=''] {
+    background: #bdc3c7 url("/blockly/media/applab/placeholder_img.png") center center no-repeat;
+  }
+
+  canvas {
```

Comment:
Nope, this only shows up in Design mode (it gets hidden when the program is run).